twisted/protocols/memcache.py

```diff
@@ -25,12 +25,15 @@ def doSomething(proto):
 more information about the protocol.
 """
 
+from __future__ import absolute_import, division
+
 from collections import deque
 
 from twisted.protocols.basic import LineReceiver
 from twisted.protocols.policies import TimeoutMixin
 from twisted.internet.defer import Deferred, fail, TimeoutError
 from twisted.python import log
+from twisted.python.compat import iteritems
```

Comment:
Can you import nativeString and networkString from twisted.python.compat?
In this file, for all changes where you did  foo.encode("ascii"), replace with networkString(foo),
and for all cases of bar.decode("ascii"), replace with nativeString(bar).

This will keep things compatible on Python 2 and Python 3